opm/parser/eclipse/EclipseState/Tables/VFPProdTable.cpp

```diff
@@ -385,6 +391,14 @@ void VFPProdTable::check() {
     }
 
     if (num_decreasing > 0) {
+        m_messages.warning("VFP table for production wells has BHP versus THP not " 
+                           + std::string("monotonically increasing.\nThis may cause convergence ")
```

Comment:
> (cannot do `"foo" + "bar"`)

True.  When we get C++14, which enables the UDL approach, this becomes more palatable.
```C++
using namespace std::string_literals;  // One of few good reasons for "using namespace"
const auto s = "This is\n"s + "one long" + " string"s;
```